Main.vb

```diff
@@ -1,6 +1,8 @@
 ﻿
 Module Test
     Public Sub Main()
+        ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls11
```

Comment:
sorry, updated the above code to include tls12, instead of referencing tls11 twice :)